### ISSUE TITLE
AUR Package

### ISSUE THREAD

**Issue by ZeroDot1** [Skywalker13/DaisyDuck#5]
Hi, is it possible to put the software in the AUR so you can easily install it in Arch Linux? 
https://aur.archlinux.org/

**Comment by Skywalker13** [comment]
Hi, I should privde an AppImage, because creating packages for all distribution is a lot of job.

**Comment by ZeroDot1** [comment]
Yes, an Appimage would be a very good idea. I would really appreciate it.
